apps/package.json

```diff
@@ -41,7 +41,7 @@
   },
   "devDependencies": {
     "@cdo/apps": "file:src",
-    "@code-dot-org/blockly": "1.6.13",
+    "@code-dot-org/blockly": "1.6.15",
```

Comment:
oops, yeah; I had to try twice, since I was logged out of npm the first time, and I didn't realize that it had got this far.

I think it's still better to leave it like this rather than trying to downgrade a version.